ucm2/sof-soundwire/RT700.conf

```diff
@@ -39,7 +40,7 @@ If.RT700 {
 			]
 
 			EnableSequence [
-
+				cset "name='DAC Front Playback Volume' 87"
```

Comment:
@plbossart speaker and headphone are using the same PCM. Do you know whether there is a kcontrol to control the sound come from, for example, mute headphone and unmute speaker?

Reply:
You'd need to rely on the jack information and use priorities to switch from one to the other.

Reply:
@plbossart Speaker and Headphone are using the same PCM. When we play with speaker, the headphone still have sound if we don't mute it. Jaroslav has a comment on RT700: https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues/854#note_471481. I think he is asking for this issue.

Reply:
@plbossart I checked the SDW platforms with RT700, and there are "Headphones Switch" and "Speaker Switch". I think we can use them in the EnableSequence/DisableSequence. I will make a patch for it.